src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/navbar-nav.html

```diff
@@ -1,3 +1,8 @@
-<ul id="navbar-main-elements" class="navbar-nav">
-    {{ generate_header_nav_html(n_links_before_dropdown=theme_header_links_before_dropdown) }}
-</ul>
+<nav class="navbar-nav">
+    <p class="sidebar-header-items__title" role="heading" aria-level="1" aria-label="{{ _('Site Navigation') }}">
+        {{ _("Site Navigation") }}
```

Comment:
is it supporting translation ?

Reply:
I am not sure exactly how Sphinx does it's translation, but that is at least the function to do so. If I need to change it to better support translation let me know how!

Reply:
my question was more, is it already part of the translated keys of Sphinx

Reply:
Do you know where this is listed? I'm not sure

Reply:
0  idea, I want to have a look at some point

Reply:
Maybe they are somewhere in here?

https://github.com/sphinx-doc/sphinx/tree/5.x/sphinx/locale

Reply:
I think that's a problem that will need to be handled elswhere (by refreshing #399) I'll approve this PR in meantime

Reply:
Makes sense - agreed we should really get that fixed up